tests/mongo-go-driver.yml

```diff
@@ -1,15 +1,15 @@
 ---
 runner: gotest
 args:
-  - -timeout=2m
-  - -shuffle=on
+  - -timeout=5m
+  - -parallel=1
```

Comment:
Let's add `-p=1` (number of packages tested in parallel; `-parallel` is for a number of tests in one package)